datasets/nldas2.data.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: nldas2
+name:  "NLDAS-2 Precipitation Forcing Dataset"
```

Comment:
this seems to be missing a description that causes data-catalog filtering to fail